homeassistant/components/stream/worker.py

```diff
@@ -164,3 +164,7 @@ def stream_worker(hass, stream, quit_event):
             # Assign the video packet to the new stream & mux
             packet.stream = buffer.vstream
             buffer.output.mux(packet)
+
+    # Close stream
+    buffer.output.close()
```

Comment:
Please either exclude this module from `.coveragerc`, or add a test.

Reply:
There are tests that cover this, but they got skipped a while back when we were transitioning from Travis to Circle and trying to find flaky tests. I unskipped and ran locally with all passing.

That being said, should we still add this to .coveragerc?

Reply:
If we want to add tests for all changes in the future it's good to check coverage for the module.

Reply:
I'm getting an error on this line:

```
File "/usr/src/homeassistant/homeassistant/components/stream/worker.py", line 169, in stream_worker
    buffer.output.close()
UnboundLocalError: local variable 'buffer' referenced before assignment
```

Obviously, the error is triggered due to a previous error:

Error demuxing stream: No dts in packet

But still the code seems wrong.